base/ip-visit-counter.yaml

```diff
@@ -64,8 +64,6 @@ spec:
           value: "http://ip-info"
         - name: IPINFOGRPCADDRESS
           value: "ip-info-grpc:5001"
-        - name: SQSQUEUENAME
-          value: "IpCount"
```

Comment:
just add it in the gke then using "patch"

Reply:
Yeah, it's already there, this is what I was saying - that's why I think it's fine to remove.

https://github.com/metalbear-co/playground/blob/e2faf26c1e71e815baeebffaa72fe0a4075ab98c/overlays/gke/kustomization.yaml#L13-L24

I wanted to make sure with you it's ok to remove, because it was added in #15, even though the patch was already there.